### PR TITLE
Pass command line arguments in context init method

### PR DESCRIPTION
**Pull request by ivanpauno**

### CODE REVIEW

File: rcljava/src/main/cpp/org_ros2_rcljava_contexts_ContextImpl.cpp [hunk 1]
```diff
@@ -98,8 +136,17 @@ Java_org_ros2_rcljava_contexts_ContextImpl_nativeDispose(JNIEnv * env, jclass, j
 
   rcl_context_t * context = reinterpret_cast<rcl_context_t *>(context_handle);
 
-  rcl_ret_t ret = rcl_context_fini(context);
+  // NOLINT, link too long
+  // TODO(ivanpauno): Currently, calling `rcl_context_fini` in a zero initialized context fails:
+  // https://github.com/ros2/rcl/blob/5b16dc99f45a85961fb86190629eeec3d9037086/rcl/src/rcl/context.c#L52-L53
+  // That's incongruent with most other rcl objects, and it shouldn't be the case.
+  // Open a bug report in `rcl`.
```
Comment (jacobperron) [inline rcljava/src/main/cpp/org_ros2_rcljava_contexts_ContextImpl.cpp]: Referencing the opened bug report here would be better for tracking if we can know to resolve this TODO.

File: rcljava/src/main/java/org/ros2/rcljava/contexts/Context.java [hunk 2]
```diff
@@ -33,10 +33,14 @@
   /**
    * Initialize the context.
    * // TODO(jacobperron): Pass arguments for parsing
```
Comment (jacobperron) [inline rcljava/src/main/java/org/ros2/rcljava/contexts/Context.java]: I think this PR resolves this TODO, not the TODO on the next line, which I was thinking about an equivalent to [`rclcpp::InitOptions`](https://github.com/ros2/rclcpp/blob/bb3debcfbaf159794298aa4096bcc04b81ef15ab/rclcpp/include/rclcpp/init_options.hpp#L28).

**Review by jacobperron — APPROVED** [review]
LGTM. I left a couple minor comments.

File: rcljava/src/main/cpp/org_ros2_rcljava_contexts_ContextImpl.cpp (hunk 1 above)
Comment (ivanpauno) [reply rcljava/src/main/cpp/org_ros2_rcljava_contexts_ContextImpl.cpp]: Done in https://github.com/osrf/ros2_java/pull/45/commits/fb1741cfd137e6a046555fe4bbb946bf4f1bb731.

File: rcljava/src/main/java/org/ros2/rcljava/contexts/Context.java (hunk 2 above)
Comment (ivanpauno) [reply rcljava/src/main/java/org/ros2/rcljava/contexts/Context.java]: Yeah, I deleted the wrong line :joy:.
Fixed in https://github.com/osrf/ros2_java/pull/45/commits/fb1741cfd137e6a046555fe4bbb946bf4f1bb731.

**Review by jacobperron — APPROVED** [review]

**Pull request merged**